articles/migrations/index.md

```diff
@@ -25,6 +25,42 @@ If you need help with the migration, create a ticket in our [Support Center](${e
 
 Current migrations are listed below, newest first. For migrations that have already been enabled see [Past Migrations](#past-migrations).
 
+### Whitelisting IP Address Ranges Australia
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Low | 2017-08-22 |  2017-09-30 |
+
+Auth0 is expanding into new global regions, and traffic originating from these regions will have new IP addresses. If you are whitelisting IP addresses, you will need to add the new addresses to your firewall rules.
+
+#### Am I affected by the change?
+
+If you are using a custom database connection, rule, and/or custom email provider that connects to your environment, **and** you have implemented firewall restrictions for IP address ranges, then you are affected by this change. You will need to make sure the following IP addresses are in your firewall rules:
```

Comment:
"the following IP address are allowed to go through your firewall"
Is this more generic? If their rules are about forbidden traffic instead of allowed the IPs being there does not help